travis.sh

```diff
@@ -60,6 +60,7 @@ function run_docker() {
         -e TRAVIS_BRANCH \
         -e TRAVIS_PULL_REQUEST \
         -e TRAVIS_OS_NAME \
+        -e TEST_PKG \
```

Comment:
This seems unrelated. Did we simply miss this param before?

Reply:
As pointed out in https://github.com/ros-planning/moveit_ci/pull/84#issuecomment-519843552, this variable needs to be passed to docker to recognize unit tests there...
So yes, we missed to pass this variable before and yes, it is related to make the unit tests pass again...